openpype/hosts/blender/api/ops.py

```diff
@@ -541,6 +541,13 @@ def register():
         update_workfile_up_to_date, first_interval=0, persistent=True
     )
 
+    # Use a timer to delay execution of check_workfile_up_to_date
+    def run_op_check_workfile_up_to_date():
```

Comment:
This method name isn't so clear imo

Reply:
What do you propose ?

Reply:
I'm not sure... Maybe something like `delayed_check_workfile_up_to_date`?